Improve Home page

e-aakash opened the pull request:
Backend changes for https://github.com/pulsejet/cerium/pull/4

Sends all forms (ids, name & response token) when requested in /api/forms end point

e-aakash committed:
Add api for getting all forms created

e-aakash committed:
Add FindOptions in collection.Find()

e-aakash committed:
Add stuct to send data to frontend

e-aakash committed:
Send response token when all forms requested

pulsejet commented on controllers/form.go:
```diff
@@ -127,3 +129,59 @@ var GetForm = func(w http.ResponseWriter, r *http.Request) {
+	type fields struct {
+		ID		int		`bson:"_id"`
+		Name 	int		`bson:"name"`
+		Token 	int		`bson:"responsetoken"`
```
This indentation is strange. Use the latest VS Code with the official golang extension and install the required modules to ensure this doesn't happen.

e-aakash replied:
Fixed, thanks for pointing out.

pulsejet commented:
You might want to query the DB to give you forms in order of descending created date, since this makes most sense practically.

e-aakash committed:
Code linting changes

e-aakash committed:
Add timestamp to form model & send sorted result

e-aakash committed:
Move Forms.Timestamp to new form creation part

Timestamp is not changed on updating the form, and stores only
time of creation of form.

e-aakash committed:
Add option to delete form & its responses

pulsejet reviewed (approved):
Awesome!
